### ISSUE TITLE
No matching distribution found for wiotp-sdk

### ISSUE THREAD

**Issue by kaizoku-619** [ibm-watson-iot/iot-python#131]
When trying to install wiotp-sdk I get the following error, even thought system is up to date and all requirements are satisfied, I'm using debian 9.

```
Collecting wiotp-sdk
  Could not find a version that satisfies the requirement wiotp-sdk (from versions: )
No matching distribution found for wiotp-sdk
```

**Comment by durera** [comment]
We’re preparing a major new release that will be the long delayed 1.0 release, once that goes out it will be released as `wiotp-sdk` but it’s not quite finished yet.

The latest release currently available in pypi is the 0.4.0 release under the old `ibmiotf` package: https://github.com/ibm-watson-iot/iot-python/tree/0.4.0

To work with `wiotp-sdk` at the moment you will need to install from git.

At this point it’s not quite ready for the 1.0 release, but I’ll endeavour to put a 0.9 `wiotp-sdk` onto pypi on Monday to make it easier to pick up the revamped library :+1:

**Comment by durera** [comment]
https://pypi.org/project/wiotp-sdk/ .. 0.9.0 available.  Note there may be minor breaking changes before the 1.0 release, but it's "nearly there".
